### ISSUE TITLE
Update the documentation

### ISSUE THREAD

**Issue by haydn** [sketchbook-js/sketchbook#85]
`README.md`, `CONTRIBUTING.md` and `LICENSE` all need updating:

- [x] Update website links to https://sketchbookjs.com
- [x] Update demo links to https://demo.sketchbookjs.com
- [x] Updated screenshot
- [ ] Update getting started instructions
- [x] Add ColonyDB sponsorship badge to README (with a link to `https://colonydb.com/?utm_source=sketchbook-js&utm_campaign=oss`)
- [x] Replace Spectrum links with GitHub discussion
- [x] Copyright year needs bumping
- [x] Add Michael to LICENSE
- [ ] Update GitHub project link

**Comment by Mxchaeltrxn** [comment]
I've made the changes in [this commit](https://github.com/sketchbook-js/sketchbook/commit/9b53e491ef456f764bc5548fbd98c370131cc5c5)

**Comment by haydn** [comment]
@Mxchaeltrxn Thanks! I've ticked of the one that have been resolved. A few notes:

- As I mentioned on the [website change](https://github.com/sketchbook-js/website/issues/2), the `yarn sketchbook start` command needs the design file passed to it.

- I noticed this links to the wrong project (I've added it to the list):

  > Take a look at the roadmap to see what's planned:
  >
  > https://github.com/sketchbook-js/sketchbook/projects/1

- We got a build failure for this commit because it made it onto `master` before the `build-command` branch was merged:
  
  https://vercel.com/sketchbook-js/sketchbook/Gr1QyshPqgHRpAMC12Kn7VbQFUya
  
  Although pushing commits straight to `main` for the `sketchbook-js/website` repo harmless, I think we should be a bit more careful in this repo and pull all changes through a PR process.
